hpr006_works-14-18/README.md

```diff
@@ -0,0 +1,39 @@
+# HPR006: Wax Mathews - Works 14-18
+
+**Release Date:** 29.03.2024
+
+## About
+Half Past Vibe Records is pleased to present Wax Mathews' newest extended play, "Works 14-18."
+
+After a year-long hiatus, and following on the heels of 2022's excellent System Ctl collaboration "Vielen Dank Rmx", we are excited to take the tempo down for our newest release. This collection of smooth, DX7-laden house groovers was composed between 2014 and 2018 on the traditional lands of the Lekwungen peoples, now known as Victoria, Canada. These tracks were recently unearthed from Wax Mathews' solid-state digital vaults - many have never been previously released. Rather than hoard them in the archives, we've re-mixed, mastered, and unit-tested these tracks, and they're now ready to be relinquished on your club stakeholders.
+
+This release also marks a bittersweet occasion - the end of the Wax Mathews alias. While this has been label co-manager James Gray's nom-de-plume for almost a decade, future releases will be under his birth name. Sharing these tracks from Wax Mathews' mid-'20s era (that's both in age and in average BPM) seems a fitting end to the chapter.
+
+## Usage
+All tracks have been stress-tested in production dancefloor sets over the past decade in various contexts, and are ready for your enjoyment.
+
+1. `Do It Again` is a dreamy 4x4 study, based around swirling DX7 chords, a powerful bassline and crisp 909 drums, with a formant-enhanced sample of Toni Braxton's "You're Makin' Me High."
+
+2. `Hurtin' Thing` is a shuffling ode to the French Touch, a layer cake built around a sample of Gloria Ann Taylor's rendition of Lou Rawls' "Love is a Hurtin' Thing", with a dash of DX7 organ and chimes, and a healthy ration of punchy drums.
+
+3. `Hydroplane City` is a straightforward, tropical nod to the 90s, with melodies and chords using classic presets from both the DX7 and Roland JD-990, and a thick Moog bassline.
+
+4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into wonky, filtered house tune inspired by the sounds of the Canadian Riviera.
```

Comment:
```suggestion
4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into a wonky, filtered house tune inspired by the sounds of the Canadian Riviera.
```

or

```suggestion
4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into wonky, filter house inspired by the sounds of the Canadian Riviera.
```

Reply:
I wouldn't call it "filter house", but I can try to tweak the grammar